Source/WebCore/page/EventHandler.cpp

```diff
@@ -4416,17 +4416,31 @@ bool EventHandler::handleTextInputEvent(const String& text, Event* underlyingEve
 
     Ref frame = m_frame.get();
 
+    auto isEnterKeypress = [underlyingEvent]()
```

Comment:
Nit: You can omit an empty parameter list.
```suggestion
    auto isEnterKeypress = [underlyingEvent]
```